brewtils/plugin.py

```diff
@@ -520,7 +534,7 @@ def _create_connection_poll_thread(self):
         connection_poll_thread.daemon = True
         return connection_poll_thread
 
-    def _invoke_command(self, target, request):
+    def _invoke_command(self, target, command, parameters):
```

Comment:
Docstring change

Reply:
Docstrings on a private function *scoff* lol I'll update it :)